libs/sdk-ui-ext/src/internal/components/configurationPanels/LineChartBasedConfigurationPanel.tsx

```diff
@@ -14,12 +15,15 @@ import {
 } from "../../constants/bubble";
 import BaseChartConfigurationPanel from "./BaseChartConfigurationPanel";
 
+export const CHARTS_SUPPORTING_DATAPOINT_CTRL = ["line", "area"];
```

Comment:
why is this enum here when it is not used in this file?

Reply:
By accident from previous implementation. Will remove it.